src/ucp/wireup/select.c

```diff
@@ -1268,6 +1269,10 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
     ucp_tl_bitmap_t tl_bitmap;
     uint8_t i;
 
+    if (ep_init_flags & UCP_EP_INIT_CM_PHASE) {
+        return UCS_OK;
+    }
```

Comment:
are these checks (here and before) unrelated fixes?

Reply:
this is related, we allow only AM lane for CM initial phase